playbooks/cloud-post.yml

```diff
@@ -37,6 +37,14 @@
     state: present
   when: cloud_instance_ip != "localhost"
 
+- name: Mount tmpfs
+  import_tasks: tmpfs/main.yml
+  when:
+    - pki_in_tmpfs
+    - not algo_store_pki
+    - ansible_system == "Darwin" or
+      ansible_system == "Linux"
```

Comment:
Can algo be used on machines that doesn't have much memory? Do people do that?

I mean, it might be that either `/dev/shm` won't exist on Linux (https://superuser.com/questions/45342/when-should-i-use-dev-shm-and-when-should-i-use-tmp) or that we won't have enough memory to use it.

Maybe we should check for those two conditions? Not sure how to determine the required memory, maybe we could skip that or just have a minimum required.

Reply:
I'll add a check to ensure that `/dev/shm` is available. Algo doesn't require much memory to store files, and rather Ansible fails if there is not enough memory available.

Reply:
Actually Ansible doesn't even run without `/dev/shm` being mounted. So, I would rather just add  these cases to the troubleshooting guide.